Tools/jsdoc/cesium_template/static/styles/jsdoc-default.css

```diff
@@ -429,6 +434,9 @@ h6
 .nameContainer .signature {
     font-weight: normal;
 }
+.nameContainer .name {
+    font-family: Consolas, "Liberation Mono", Menlo, "Courier New", Courier, Monaco, monospace;
```

Comment:
I'm late to the party, but it might be nice to consolidate these at the top:

```
#ClassList > li,
.page-title,
nameContainer .name {
    font-family: Consolas, "Liberation Mono", Menlo, "Courier New", Courier, Monaco, monospace;
}
```

You can still have a separate .page-title section later on.

Just a DRY thought, no need to spin wheels on this.